src/screens/SavedEventListScreen/component.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import React, { PureComponent } from "react";
+import { StyleSheet, View } from "react-native";
+import type { NavigationScreenProp, NavigationState } from "react-navigation";
+import type { SavedEvents, EventDays } from "../../data/event";
+import type { LocalizedFieldRef } from "../../data/localized-field-ref";
+import EventList from "../../components/EventList";
+import text from "../../constants/text";
+import StandaloneHeader from "../../components/StandaloneHeader";
+import Loading from "../../components/Loading";
+import { bgColor } from "../../constants/colors";
+import { EVENT_DETAILS } from "../../constants/routes";
+import locale from "../../data/locale";
+
+export type Props = {
+  navigation: NavigationScreenProp<NavigationState>,
+  events: EventDays,
+  savedEvents: SavedEvents,
+  addSavedEvent: string => void,
+  removeSavedEvent: string => void,
+  loading: boolean,
+  refreshing: boolean,
+  updateEvents: () => Promise<void>,
+  getAssetUrl: LocalizedFieldRef => string
+};
+
+class SavedEventListScreen extends PureComponent<Props> {
+  static navigationOptions = {
+    header: null
+  };
+
+  render() {
+    const {
+      navigation,
+      updateEvents,
+      events,
+      savedEvents,
+      addSavedEvent,
+      removeSavedEvent,
+      refreshing,
+      getAssetUrl
+    } = this.props;
+    return (
+      <View style={styles.container}>
+        <StandaloneHeader title={text.savedEventsTitle} />
+        {this.props.loading ? (
+          <Loading />
+        ) : (
+          <EventList
+            locale={locale}
+            events={events}
+            savedEvents={savedEvents}
+            addSavedEvent={addSavedEvent}
+            removeSavedEvent={removeSavedEvent}
+            refreshing={refreshing}
+            onRefresh={() => {
+              updateEvents();
```

Comment:
You might be able to simplify this by just passing `updateEvents` instead of wrapping in an arrow. I.e.

```
onRefresh={updateEvents}
```